clang-tools-extra/docs/clang-tidy/checks/readability/use-std-min-max.rst

```diff
@@ -0,0 +1,32 @@
+.. title:: clang-tidy - readability-use-std-min-max
+
+readability-use-std-min-max
+===========================
+
+Replaces certain conditional statements with equivalent ``std::min`` or
+``std::max`` expressions. Note: This may impact
+performance in critical code due to potential additional stores compared
+to the original if statement.
+
+
+Examples:
```

Comment:
Not needed, remove duplicated empty line and 'Examples:', Before and after should be fine